src/main/java/org/terasology/scenario/internal/systems/ConvertEntitySystem.java

```diff
@@ -77,14 +82,12 @@
 
     private Logger logger = LoggerFactory.getLogger(ConvertEntitySystem.class);
 
-
-    @ReceiveEvent
-    public void onConvertEntityEvent(ConvertScenarioEntityEvent event, EntityRef entity, ScenarioValueTriggeringRegionComponent component) {
-        event.getOutputList().add(event.getPrefix() + PREFAB_MARKER + entity.getParentPrefab().getName());
-    }
-
-    @ReceiveEvent
-    public void onConvertEntityEvent(ConvertScenarioEntityEvent event, EntityRef entity, ScenarioSecondarySendChatComponent component) {
+    /**
+     * Anything that has an argument container and is not a value would use this serialization
+     * @param event the ConvertScenarioEntityEvent that triggered the original call
+     * @param entity The entityRef that the call was sent to
+     */
+    public void default_serialize(ConvertScenarioEntityEvent event, EntityRef entity) {

```

Comment:
Nice code duplication elimination

However please use proper camel case for method name as it is common in java